imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -280,24 +138,8 @@
    },
    "outputs": [],
    "source": [
-    "# Path to export DICOM data.\n",
-    "dicom_store_url = os.path.join(HEALTHCARE_API_URL, 'projects', project_id, 'locations', location, 'datasets', dataset_id, 'dicomStores', dicom_store_id)\n",
-    "path = dicom_store_url + \":export\"\n",
-    "\n",
-    "# Headers (send request in JSON format).\n",
-    "headers = {'Content-Type': 'application/json'}\n",
-    "\n",
-    "# Body (encoded in JSON format).\n",
-    "output_config = {'output_config': {'gcs_destination': {'uri_prefix': jpeg_folder, 'mime_type': 'image/jpeg; transfer-syntax=1.2.840.10008.1.2.4.50'}}}\n",
-    "body = json.dumps(output_config)\n",
-    "\n",
-    "resp, content = http.request(path, method='POST', headers=headers, body=body)\n",
-    "assert resp.status == 200, 'error exporting to JPEG, code: {0}, response: {1}'.format(resp.status, content)\n",
-    "print('Full response:\\n{0}'.format(content.decode()))\n",
-    "\n",
-    "# Record operation_name so we can poll for it later.\n",
-    "response = json.loads(content.decode())\n",
-    "operation_name = response['name']"
+    "%%bash -s {jpeg_folder}\n",
+    "gcloud beta healthcare --project chc-tcia  dicom-stores export gcs cbis-ddsm --location=us-central1 --dataset=cbis-ddsm --mime-type=\"image/jpeg; transfer-syntax=1.2.840.10008.1.2.4.50\" --gcs-uri-prefix=$1"
```

Comment:
Do we need to wait until this call is completed?

Reply:
yes, this action holds executions until export is done